### PR TITLE
[CP] More 3.17 RC2 cps and version bump

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
[test katello]

File: app/services/katello/pulp3/migration.rb [hunk 1]
```diff
@@ -3,20 +3,27 @@
...
 
-      def initialize(smart_proxy, repository_types = REPOSITORY_TYPES)
+      UNIFIED_CONTENT_TYPES = [
+        Katello::Erratum
+      ].freeze
+
+      def self.repository_types_for_migration
+        #we can migrate types that pulp3 supports, but are overridden to pulp2.  These are in 'migration mode'
+        overridden = (SETTINGS[:katello][:use_pulp_2_for_content_type] || {}).keys.select { |key| SETTINGS[:katello][:use_pulp_2_for_content_type][key] }
+        overridden.select { |type| SmartProxy.pulp_master.pulp3_repository_type_support?(type.to_s, false) }.map { |t| t.to_s }
```
Comment (ianballou) [inline app/services/katello/pulp3/migration.rb:21]: Looks like `master` snuck in to replace `primary` for the yum migration commits.

**Comment by chris1984** [comment]
@jlsherrill that looks better, 4 tests are still failing

**Comment by chris1984** [comment]
[test katello]

**Comment by chris1984** [comment]
@ianballou tests are green, can you review again?

**Review by ianballou — APPROVED** [review]

**Pull request merged**